src/style.css

```diff
@@ -1 +1,21 @@
-/* styles go here */
+/* stylelint-disable selector-type-no-unknown */
+/* stylelint-disable block-no-empty */
+/* stylelint-disable no-duplicate-selectors */
+/* stylelint-disable block-opening-brace-space-before */
+/* stylelint-disable rule-empty-line-before */
+/* stylelint-disable indentation */
```

Comment:
Don't disable the eslint. Try to fix its problems

Reply:
How?